Source/DataSources/ModelVisualizer.js

```diff
@@ -176,6 +179,16 @@ define([
                     modelData.animationsRunning = runAnimations;
                 }
 
+                // Update the speedup property
+                if (modelData.animationsRunning && modelData.speedup !== speedup) {
+                    var animationLength = model.activeAnimations.length;
+                    for (var animationIndex = 0; animationIndex < animationLength; ++animationIndex) {
```

Comment:
Rename `animationIndex` to `i` for simplicity and readability

Reply:
I was following the style a few lines below here: https://github.com/AnalyticalGraphicsInc/cesium/blob/7b36978f16d55ae7c290047e79b24ff212714474/Source/DataSources/ModelVisualizer.js#L197-L198

I think `i` is already defined in the outer loop so I'll name it `j`.